CHANGELOG.md

```diff
@@ -4,16 +4,17 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.1.0]
```

Comment:
Keep the `Unreleased` header and the related diff link?

Reply:
I was just going to add it back in with the next changelog entry, since it kind of looks odd to have Unreleased at the top of a release's changelog.